pillar.example

```diff
@@ -6,6 +6,7 @@ nginx:
   # This values can be overwritten in each server context individually.
   ssl_protocols: 'TLSv1.2 TLSv1.3'
   ssl_ciphers: 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256' # noqa: 204
+  ssl_conf_command: []
```

Comment:
I think you should add a "real" example on how to use this, but mention the default is empty (maybe in a seperate paragraph), then test for that change to actually be in the file in the test below.